src/bzflag/bzflag.cxx

```diff
@@ -1288,7 +1288,7 @@ int         main(int argc, char** argv)
             case GL_OUT_OF_MEMORY:
                 std::cerr << "ERROR: GL_OUT_OF_MEMORY" << std::endl;
                 break;
-#ifdef GL_VERSION_1_2
+#ifdef GL_TABLE_TOO_LARGE
```

Comment:
is this optional in GL 2.1? If not, we can probably do away with the `#ifdef`

Reply:
That is eventually for GLES. GL_TABLE_TOO_LARGE is not applicable to it

Reply:
When that time comes, we should find a better test than that (and there will need to be many others). Compiler errors will show any unsupported code that needs to be updated. In my GLES port, I just defined `HAVE_GLES`, which also made it easy to find any GLES-specific code. I suggest just remove that `#ifdef` for now.

Reply:
lot of code out of here is using that ifdef.
But I don't care removing that for now